src/compiler/moduleNameResolver.ts

```diff
@@ -2140,7 +2158,7 @@ export function getEntrypointsFromPackageJsonInfo(
 function loadEntrypointsFromExportMap(
     scope: PackageJsonInfo,
     exports: object,
-    state: ModuleResolutionState,
+    state: ModuleResolutionState & { host: GetPackageJsonEntrypointsHost },

```

Comment:
I was going to ask if this would be better served as a parameter rather than tacked onto the state, but it does seem like this was already happening given what's in `useCaseSensitiveFileNames`.

Reply:
This isn’t adding a `host` property, it’s narrowing the type of the existing `host` property.

Reply:
Yeah, I was just unsure without trying to find this type (which I should have done!)